### DISCUSSION TITLE
I want landmark navigation to be implemented natively in the browsers

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#63] [Wants]
Certain HTML structural elements: `main`, `nav`, `header` (in `body` scope), `footer` (in `body` scope), `aside`, `form` (with accessible label), and `section` (with accessible label) can be used to organize the content of a web page or web app into landmark regions. If you are a screen reader user, you can take advantage of a well-structured page by using keyboard shortcuts to navigate directly to landmark content. This is more efficient in terms of the number of keystrokes (i.e., it may have taken many tab key presses to move focus to the same place). Landmarks also give screen reader users a high-level overview of the page content.

However, if you are not a screen reader user, you cannot take advantage of this capability. In fact, you won't even know those landmarks are there (unless you install a browser extension).

I have opened or commented on requests to have landmark navigation for all users added to each of the mainstream browsers:

- Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=670928#c70
- Chrome/Edge: https://bugs.chromium.org/p/chromium/issues/detail?id=704698#c23
- Safari: https://bugs.webkit.org/show_bug.cgi?id=172817#c2

I think this would be a really useful feature that would improve the web experience for anyone who uses a keyboard.

<hr>

https://webwewant.fyi/wants/5f24adc28dfea402c3830218/

**Comment by dfarough** [comment]
I agree! More Keyboard support in browsers would help a lot.

**Comment by benbeaudry** [comment]
Agreed! I've been hearing this ask from so many different sources over the years, and today many websites implement this as a custom feature with a unique keyboard shortcut, which makes it hard to use consistently.

I'm reopening discussions on this in [this issue](https://github.com/openui/open-ui/issues/1472) of the OpenUI group, and wrote [a (very early) draft explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/Accessibility/FocusLandmark/explainer.md) that could solve this. Please, share feedback!
